Src/Microsoft.Dynamic/ComInterop/ComRuntimeHelpers.cs

```diff
@@ -659,14 +667,13 @@ out argErr
                     typeof(IntPtr), // excepInfo
                     typeof(IntPtr), // argErr
                 };
-            signature.AddArguments(invokeParamTypes, null, null);
-            method.Emit(OpCodes.Calli, signature);
+            method.EmitCalli(OpCodes.Calli, CallingConvention.StdCall, typeof(int), invokeParamTypes);
```

Comment:
Any reason for the change of calling convention here?

Reply:
I was debugging something at the time. I'll change it back to `WinApi`.